data-prepper-api/src/main/java/org/opensearch/dataprepper/model/metric/JacksonSum.java

```diff
@@ -102,18 +102,27 @@ public Builder withIsMonotonic(final boolean isMonotonic) {
         }
 
         /**
-         * Returns a newly created
+         * Returns a newly created {@link JacksonSum}
          * @return a JacksonSum
          * @since 1.4
          */
         public JacksonSum build() {
+            return build(true);
```

Comment:
Do you think it would make sense to introduce a constant somewhere in order to name this parameter so it would make it easier for the reader to understand the meaning of it ?

Reply:
I would be more in favor of having this parameter be a part of the `.Builder` classes as well